lib/utils/log/slog_text_handler_darwin.go

```diff
@@ -36,7 +37,7 @@ func NewSlogOSLogHandler(subsystem string, level slog.Leveler) (*SlogTextHandler
 			// timestamp fields in the message. os_log has dedicated handling for this kind of metadata.
 			ConfiguredFields: []string{CallerField},
 		},
-		out:        newOSLogWriter(subsystem),
+		out:        w,
```

Comment:
The changes to this file seem unnecessary unless we're planning to do more with `w`. Could this be removed to preserve the history?